src/main/java/net/fabricmc/loom/build/mixin/KaptApInvoker.java

```diff
@@ -62,18 +64,26 @@ public KaptApInvoker(Project project) {
 		kaptExtension.setIncludeCompileClasspath(false);
 	}
 
+	private static Map<SourceSet, JavaCompile> getInvokerTasks(Project project) {
+		MixinAnnotationProcessorExtension mixin = project.getExtensions().getByType(MixinAnnotationProcessorExtension.class);
+		return mixin.getInvokerTasks(project, AnnotationProcessorInvoker.JAVA)
```

Comment:
Question: Should this be java or kotlin?

Reply:
The old code uses Java. I basically copy & paste it here.

Reply:
Thats fine, I should have checked, was just questioning it.